packages/design-system/src/styles/components/_FilterChip.scss

```diff
@@ -81,12 +81,6 @@ $filter-chip-icon-size: 10px;
   .ds-c-icon--close-thin {
     height: $filter-chip-icon-size + 2;
     width: $filter-chip-icon-size + 2;
-
```

Comment:
Found this style was unnecessary as `.ds-c-icon--close` has the same styles applied and the thin icon also received those styles as well. 

tldr; redundant styling.